sdk/test-utils/recorder/ASSET_SYNC_MIGRATION.md

```diff
@@ -1,23 +1,26 @@
 # Migrating recordings to the `azure-sdk-assets` repository
 
-## Some background
+# Background
 
 Recordings take up a large amount of space in our repository and generate a lot of churn. The asset sync project, owned by our Engineering System team, is a solution to this problem. Leveraging and extending the existing test proxy, the asset sync extension adds support for moving the recordings outside of the main azure-sdk-for-js repo.
 
 ## Performing the migration
 
 ### Prerequisites
 
```

Comment:
Do we need to add guide on the permission of asset repo?

Reply:
Added that too, will remove if someone objects